Add note about XNB (#1)
Lekuruu/osu-modding: README.md

You now have the option to right-click on a resource and "Save", or even edit the resource entirely by clicking "Edit Resource..."

> ### XNB
> In builds before ~b20130606, osu! assets will be stored as XNB files, which is a file format made for the XNA framework (what osu! uses).

Reviewer: ```suggestion
### XNB

In builds before ~b20130606, osu! assets will be stored as XNB files, which is a file format made for the XNA framework (what osu! uses).
```

*s p a c e*

Author: looks

good

> This means that you'll need to convert them from XNB to PNG files, which Levi has a tool for [here](https://github.com/lekuruu/xnb)

Reviewer: ```suggestion
This means that you'll need to convert them from XNB to PNG files, which I have a tool for [here](https://github.com/lekuruu/xnb).
```

I used "I" in this document everywhere, so lets continue that for now
